rest/admin_api_auth_test.go

```diff
@@ -345,7 +345,10 @@ func TestAdminAuth(t *testing.T) {
 	if base.UnitTestUrlIsWalrus() {
 		t.Skip("Test requires Couchbase Server")
 	}
-
+	serverURL := base.UnitTestUrl()
+	if base.ServerIsTLS(serverURL) {
+		t.Skipf("URI %s needs to start with couchbase://", serverURL)
+	}
 	rt := NewRestTester(t, nil)
```

Comment:
This test can be made to work with either scheme. Preferable to skipping.

Potentially similar fix for below tests, although have not looked to see why those are failing.

```suggestion
	rt := NewRestTester(t, &RestTesterConfig{useTLSServer: base.ServerIsTLS(base.UnitTestUrl())})
```

Reply:
there were so many circumstances where RestTesterConfig is initialized, so I moved the logic to set TLS in NewRestTester. All integration tests seem to pass.